### PR TITLE
rename tickersKey to currencyKey in CoinTicker

### PR DESCRIPTION
**Pull request by johnnynanjiang**

### CODE REVIEW

File: Trust/Core/Initializers/MirgrationInitializer.swift [hunk 1]
```diff
@@ -36,9 +36,18 @@ class MigrationInitializer: Initializer {
...
             default:
                 break
             }
         }
     }
 }
+
+class RealmMigration {
+    class func renameTickerCurrencyKey(_ migration: Migration) {
+        migration.renameProperty(onType: "CoinTicker", from: "tickersKey", to: "currencyKey")
```
Comment (vikmeup) [inline Trust/Core/Initializers/MirgrationInitializer.swift]: can you just delete `CoinTicker ` model for simplicity?

File: Trust/Core/Initializers/MirgrationInitializer.swift [hunk 2]
```diff
@@ -36,9 +36,18 @@ class MigrationInitializer: Initializer {
                 fallthrough
             case 33...49:
                 migration.deleteData(forType: Transaction.className)
+                fallthrough
+            case 50...52:
+                RealmMigration.renameTickerCurrencyKey(migration)
```
Comment (vikmeup) [inline Trust/Core/Initializers/MirgrationInitializer.swift]: you need to `fallthrough`

**Comment by vikmeup** [comment]
I don't think we need to rename at all.

**Comment by johnnynanjiang** [comment]
I thought it would be ideal to rename tickersKey to currencyKey to avoid confusion, but it is ok to leave it as is.

**Pull request closed without merging**